client/src/components/moviesList/movieBlock/movieBlock.component.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styles from './movieBlock.css';
+
+export default function createMovieBlock(data, onMovieClick) {
+
+  const onClick = (data) => {
```

Comment:
This function is redundant since you can pass props.onMovieClick directly to onClick
() => props.onMovieClick(data)